ui/helpers/constants/common.ts

```diff
@@ -23,3 +23,24 @@ export const OUTDATED_BROWSER_VERSIONS = {
   // See https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser
   opera: '<76',
 };
+export const BROKEN_PRERENDER_BROWSER_VERSIONS = {
```

Comment:
Nit: Probably worthwhile having at least brief descriptions of these constants as well

Reply:
agreed https://github.com/MetaMask/metamask-extension/pull/24317/commits/08279efdf8ad998044bae5cd06f1d46e99ddee2a